--- a/IceCubesApp/App/Tabs/Settings/AddAccountsView.swift
+++ b/IceCubesApp/App/Tabs/Settings/AddAccountsView.swift
@@ -57,1 +57,1 @@
+    NavigationSplitView { NavigationStack {
-    NavigationStack {
@@ -159,1 +159,1 @@
+  } } detail: { Color.clear }
-  }
--- a/IceCubesApp/App/Tabs/TagGroup/EditTagGroupView.swift
+++ b/IceCubesApp/App/Tabs/TagGroup/EditTagGroupView.swift
@@ -25,1 +25,1 @@
+    NavigationSplitView { NavigationStack {
-    NavigationStack {
@@ -79,1 +79,1 @@
+  } } detail: { Color.clear }
-  }
--- a/IceCubesApp/App/Tabs/NavigationSheet.swift
+++ b/IceCubesApp/App/Tabs/NavigationSheet.swift
@@ -17,1 +17,1 @@
+    NavigationSplitView { NavigationStack {
-    NavigationStack {
@@ -23,1 +23,1 @@
+  } } detail: { Color.clear }
-  }
--- a/IceCubesApp/App/Tabs/Timeline/AddRemoteTimelineView.swift
+++ b/IceCubesApp/App/Tabs/Timeline/AddRemoteTimelineView.swift
@@ -26,1 +26,1 @@
+    NavigationSplitView { NavigationStack {
-    NavigationStack {
@@ -88,1 +88,1 @@
+  } } detail: { Color.clear }
-  }
--- a/IceCubesApp/App/Report/ReportView.swift
+++ b/IceCubesApp/App/Report/ReportView.swift
@@ -19,1 +19,1 @@
+    NavigationSplitView { NavigationStack {
-    NavigationStack {
@@ -69,1 +69,1 @@
+  } } detail: { Color.clear }
-  }
--- a/Packages/MediaUI/Sources/MediaUI/MediaUIView.swift
+++ b/Packages/MediaUI/Sources/MediaUI/MediaUIView.swift
@@ -15,1 +15,1 @@
+    NavigationSplitView { NavigationStack {
-    NavigationStack {
@@ -57,1 +57,1 @@
+  } } detail: { Color.clear }
-  }
--- a/Packages/MediaUI/Sources/MediaUI/MediaUIAttachmentVideoView.swift
+++ b/Packages/MediaUI/Sources/MediaUI/MediaUIAttachmentVideoView.swift
@@ -154,1 +154,1 @@
+    NavigationSplitView { NavigationStack {
-    NavigationStack {
@@ -195,1 +195,1 @@
+  } } detail: { Color.clear }
-  }
--- a/Packages/Lists/Sources/Lists/AddAccounts/ListAddAccountView.swift
+++ b/Packages/Lists/Sources/Lists/AddAccounts/ListAddAccountView.swift
@@ -21,1 +21,1 @@
+    NavigationSplitView { NavigationStack {
-    NavigationStack {
@@ -75,1 +75,1 @@
+  } } detail: { Color.clear }
-  }
--- a/Packages/Lists/Sources/Lists/Edit/ListEditView.swift
+++ b/Packages/Lists/Sources/Lists/Edit/ListEditView.swift
@@ -21,1 +21,1 @@
+    NavigationSplitView { NavigationStack {
-    NavigationStack {
@@ -111,1 +111,1 @@
+  } } detail: { Color.clear }
-  }
--- a/Packages/Lists/Sources/Lists/Create/ListCreateView.swift
+++ b/Packages/Lists/Sources/Lists/Create/ListCreateView.swift
@@ -23,1 +23,1 @@
+    NavigationSplitView { NavigationStack {
-    NavigationStack {
@@ -70,1 +70,1 @@
+  } } detail: { Color.clear }
-  }
--- a/Packages/AppAccount/Sources/AppAccount/AppAccountsSelectorView.swift
+++ b/Packages/AppAccount/Sources/AppAccount/AppAccountsSelectorView.swift
@@ -129,1 +129,1 @@
+    NavigationSplitView { NavigationStack {
-    NavigationStack {
@@ -181,1 +181,1 @@
+  } } detail: { Color.clear }
-  }
--- a/Packages/Account/Sources/Account/Filters/FiltersListView.swift
+++ b/Packages/Account/Sources/Account/Filters/FiltersListView.swift
@@ -21,1 +21,1 @@
+    NavigationSplitView { NavigationStack {
-    NavigationStack {
@@ -90,1 +90,1 @@
+  } } detail: { Color.clear }
-  }
--- a/Packages/Account/Sources/Account/Edit/EditRelationshipNoteView.swift
+++ b/Packages/Account/Sources/Account/Edit/EditRelationshipNoteView.swift
@@ -21,1 +21,1 @@
+    NavigationSplitView { NavigationStack {
-    NavigationStack {
@@ -53,1 +53,1 @@
+  } } detail: { Color.clear }
-  }
--- a/Packages/Account/Sources/Account/Edit/EditAccountView.swift
+++ b/Packages/Account/Sources/Account/Edit/EditAccountView.swift
@@ -20,1 +20,1 @@
+    NavigationSplitView { NavigationStack {
-    NavigationStack {
@@ -55,1 +55,1 @@
+  } } detail: { Color.clear }
-  }
--- a/Packages/Timeline/Sources/Timeline/View/TimelineContentFilterView.swift
+++ b/Packages/Timeline/Sources/Timeline/View/TimelineContentFilterView.swift
@@ -18,1 +18,1 @@
+    NavigationSplitView { NavigationStack {
-    NavigationStack {
@@ -65,1 +65,1 @@
+  } } detail: { Color.clear }
-  }
--- a/Packages/Notifications/Sources/Notifications/List/NotificationsPolicyView.swift
+++ b/Packages/Notifications/Sources/Notifications/List/NotificationsPolicyView.swift
@@ -15,1 +15,1 @@
+    NavigationSplitView { NavigationStack {
-    NavigationStack {
@@ -125,1 +125,1 @@
+  } } detail: { Color.clear }
-  }
--- a/Packages/StatusKit/Sources/StatusKit/History/StatusEditHistoryView.swift
+++ b/Packages/StatusKit/Sources/StatusKit/History/StatusEditHistoryView.swift
@@ -21,1 +21,1 @@
+    NavigationSplitView { NavigationStack {
-    NavigationStack {
@@ -64,1 +64,1 @@
+  } } detail: { Color.clear }
-  }
--- a/Packages/StatusKit/Sources/StatusKit/Editor/Drafts/DraftsListView.swift
+++ b/Packages/StatusKit/Sources/StatusKit/Editor/Drafts/DraftsListView.swift
@@ -18,1 +18,1 @@
+      NavigationSplitView { NavigationStack {
-      NavigationStack {
@@ -57,1 +57,1 @@
+    } } detail: { Color.clear }
-    }
--- a/Packages/StatusKit/Sources/StatusKit/Editor/Components/MediaEditView.swift
+++ b/Packages/StatusKit/Sources/StatusKit/Editor/Components/MediaEditView.swift
@@ -30,1 +30,1 @@
+      NavigationSplitView { NavigationStack {
-      NavigationStack {
@@ -127,1 +127,1 @@
+    } } detail: { Color.clear }
-    }
--- a/Packages/StatusKit/Sources/StatusKit/Editor/Components/CustomEmojisView.swift
+++ b/Packages/StatusKit/Sources/StatusKit/Editor/Components/CustomEmojisView.swift
@@ -17,1 +17,1 @@
+      NavigationSplitView { NavigationStack {
-      NavigationStack {
@@ -63,1 +63,1 @@
+    } } detail: { Color.clear }
-    }
--- a/Packages/StatusKit/Sources/StatusKit/Editor/Components/LanguageSheetView.swift
+++ b/Packages/StatusKit/Sources/StatusKit/Editor/Components/LanguageSheetView.swift
@@ -18,1 +18,1 @@
+      NavigationSplitView { NavigationStack {
-      NavigationStack {
@@ -42,1 +42,1 @@
+    } } detail: { Color.clear }
-    }
--- a/Packages/StatusKit/Sources/StatusKit/Editor/MainView.swift
+++ b/Packages/StatusKit/Sources/StatusKit/Editor/MainView.swift
@@ -47,1 +47,1 @@
+      NavigationSplitView { NavigationStack {
-      NavigationStack {
@@ -55,1 +55,1 @@
+    } } detail: { Color.clear }
-    }
--- a/Packages/StatusKit/Sources/StatusKit/Share/StatusRowShareAsImageView.swift
+++ b/Packages/StatusKit/Sources/StatusKit/Share/StatusRowShareAsImageView.swift
@@ -18,1 +18,1 @@
+    NavigationSplitView { NavigationStack {
-    NavigationStack {
@@ -56,1 +56,1 @@
+  } } detail: { Color.clear }
-  }
--- a/Packages/StatusKit/Sources/StatusKit/Share/StatusRowSelectableTextView.swift
+++ b/Packages/StatusKit/Sources/StatusKit/Share/StatusRowSelectableTextView.swift
@@ -11,1 +11,1 @@
+    NavigationSplitView { NavigationStack {
-    NavigationStack {
@@ -27,1 +27,1 @@
+  } } detail: { Color.clear }
-  }
